README.md

```diff
@@ -34,6 +33,45 @@ Special notes:
  * Since we cannot distribute Java Richards, a patch is required to derive the
    tracing version (`patches/trace_java_richards.diff`)
 
+## Using the scripts here to analyse an existing benchmark
+
+### Requirements
```

Comment:
I think we need to sub-split this into "mandatory requirements" and "optional" requirements. At the moment, I don't think anyone will understand why there's "python modulzes" and "python modules required for plotting" as separate items.